bin/tblextr.py

```diff
@@ -59,6 +60,48 @@
 var_table = {}
 #############################################################
 
+def json_metadata_gen(sysinfo_file, index):
+    if not re.search(r'\.txt$', sysinfo_file):
+      raise Exception('wrong output file type: "' + sysinfo_file + '"' )
+    if index == 1:
+      status, output = commands.getstatusoutput("/opt/rocm/bin/rocminfo > " + sysinfo_file)
+      if status != 0 :
+        raise Exception('Could not run command: rocminfo')
+      params = gen_params(sysinfo_file);
+    elif index == 2:
+      status, output = commands.getstatusoutput("/opt/rocm/bin/hipcc --version >" + sysinfo_file)
+      if status != 0 :
+        raise Exception('Could not run command: hipcc --version')
+      params = gen_params(sysinfo_file);
+    return params
+
+def json_metadata_write(jsonfile, params, params2):
```

Comment:
Please move/rename 'json_metadata_write' it to sqlitedb.py as a 'metadata_json()' method of SQLiteDB and consume just one file parameter.
Sp to call it from tblextr.py as:
db.metadata_json(sysinfo_file)

Reply:
Done